Lib/functools.py

```diff
@@ -1042,3 +1037,24 @@ def __get__(self, instance, owner=None):
         return val
 
     __class_getitem__ = classmethod(GenericAlias)
+
+def _warn_kwargs(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if kwargs:
+            import os
+            import warnings
+            warnings.warn("""\
+Calling functools.reduce with keyword arguments is deprecated
+in Python 3.14 and will be forbidden in Python 3.16.""",
```

Comment:
Would it be possible to avoid the trailing \ and indent the string instead?

Reply:
Done.